mapdata/WarcraftLegacies/AbilityData/Skin/859000916.json

```
@@ -0,0 +1,88 @@
+{
+  "OldId": 1718764353,
+  "NewId": 859000916,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1953653089,
+      "Type": 3,
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNFelFlameStorm.blp"
+    },
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": ""
+    },
+    {
+      "Id": 2037082209,
+      "Type": 3,
+      "Value": "Q"
+    },
+    {
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Calls 20 waves off Felfire down from the sky igniting and damaging all enemy units in an area, Each wave deals 100  damage; units effected by Felfire take 25 damage per second for 4 seconds."
```

```suggestion
      "Value": "Calls 20 waves of Felfire down from the sky, damaging and igniting all enemy units in an area. Each wave deals 100 damage and causes affected units to take 25 damage per second for 4 seconds."
```
Some grammar fixes, some stylistic changes.